language/phpmailer.lang-mg.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* PHPMailer language file: refer to English translation for definitive list
+* French Version
```

Comment:
Hey @octavebenil, it's great you're still around - any chance you can finish this off and then I'll merge it.